airbyte-integrations/connectors/source-paypal-transaction/pyproject.toml

```
@@ -3,10 +3,10 @@ requires = [ "poetry-core>=1.0.0",]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
-version = "2.2.2"
+version = "2.3.0"
 name = "source-paypal-transaction"
 description = "Source implementation for Paypal Transaction."
-authors = [ "Airbyte <contact@airbyte.io>",]
+authors = [ "Airbyte <jose.pineda@airbyte.io>",]
```

```suggestion
authors = [ "Airbyte <contact@airbyte.io>",]
```

Can we keep the email address the same for consistency please?